### DISCUSSION TITLE
SyslogLocalSink/LocalSyslogService on Windows

### DISCUSSION THREAD

**Discussion by gandhiz** [IonxSolutions/serilog-sinks-syslog#52] [Q&A] [answered]
Hi, 
Would you know if the SyslogLocalSink/LocalSyslogService would work on Windows with a local syslog installed like syslog-ng?
Thanks.

**Comment by colin-anderson** [comment, marked as answer]
Yes, it will work with any syslog server that correctly implements RFC3164 or RFC5424.
